src/care/care_inst.h

```diff
@@ -41,6 +41,10 @@
 #define CARE_TEMPLATE_KEY_TYPE int
 #define CARE_TEMPLATE_ARRAY_TYPE globalID
 #include "care/KeyValueSorter_inst.h"
+
```

Comment:
Don't you need to protect this for whether GID64 is enabled? Otherwise GIDTYPE resolves to int and you are defining <int,int> twice.

Reply:
The endif on line 48 is for #if CARE_HAVE_LLNL_GLOBALID

Reply:
I think Peter may be right that the non-gid64 variant is going to break with this change (CARE_HAVE_LLNL_GLOBALID is defined in both cases, just GIDTYPE changes between int and long). If this does break, you could just instantiate both int and long.

Reply:
Oh, I get it.  I can add the protection.